### DISCUSSION TITLE
transmission-remote settings file?

### DISCUSSION THREAD

**Discussion by merlinstardust** [transmission/transmission#1947] [Q&A]
The current config page lists files for client, daemon, and CLI. It lists nothing for remote/web.

I discovered how to set remote/web settings with `transmission-remote` but they get overwritten if I restart Transmission.

Is there a remote/web config file I can use to permanently set these?

**Comment by sio** [comment]
Edit settings.json for the daemon. It is  also the backend for the web interface.

**Comment by merlinstardust** [comment]
I did edit settings.json but it did not affect the web interface. I stopped the daemon, used the necessary commands to edit settings (specifically default download and incomplete folders), and started up the daemon. But the web interface was unaffected.

The web interface was only affected when I ran the necessary commands with `transmission-web`
